docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    =
+SPHINXOPTS    = -W
```

Comment:
I would not add it here, because when making docs locally I prefer to ignore some warnings if needed